addons/docs/src/frameworks/vue3/extractArgTypes.ts

```diff
@@ -33,6 +33,13 @@ export const extractArgTypes: ArgTypesExtractor = (component) => {
           category: section,
         },
       };
+
+      // split props, slots etc which could clash on name
+      if (results[section] === undefined) {
+        // add non-enumerable property for section
+        Object.defineProperty(results, section, { value: {}, enumerable: false });
+      }
+      results[section][name] = results[name];
```

Comment:
What happens if there is a `prop` called `"slots"`?

Reply:
Good point and I guess depending on processing order a slot called 'props'.

Perhaps with the switch 'scopedArgs' the roll-up of props does not happen or goes into `results.all` as a fallback.

Not sure I like that as a solution, but I haven't thought of a better one just yet. 

Using non-permitted attribute name characters is another possibility, but I don't know enough about other frameworks to know where the clashes might be. `scopedArgs: true` could become `scopedArgs: { prefix: '$' }`

Reply:
@fallemand are you saying we should not merge this PR as-is? Or that we should.

Sorry 😅 I don't quite understand.

Reply:
@ndelangen I'm saying we SHOULD merge it 🙏 .  
The issue is a nightmare. It's a common pattern in Vue design systems to use slots and props with the same name.